crates/sui-bridge/src/config.rs

```diff
@@ -22,52 +24,49 @@ use sui_sdk::apis::CoinReadApi;
 use sui_sdk::{SuiClient as SuiSdkClient, SuiClientBuilder};
 use sui_types::base_types::ObjectRef;
 use sui_types::base_types::{ObjectID, SuiAddress};
+use sui_types::bridge::BridgeChainId;
 use sui_types::crypto::SuiKeyPair;
+use sui_types::digests::{get_mainnet_chain_identifier, get_testnet_chain_identifier};
 use sui_types::event::EventID;
 use sui_types::object::Owner;
 use tracing::info;
 
 #[serde_as]
 #[derive(Clone, Debug, Deserialize, Serialize)]
 #[serde(rename_all = "kebab-case")]
-pub struct BridgeNodeConfig {
-    /// The port that the server listens on.
-    pub server_listen_port: u16,
-    /// The port that for metrics server.
-    pub metrics_port: u16,
-    /// Path of the file where bridge authority key (Secp256k1) is stored as Base64 encoded `privkey`.
-    pub bridge_authority_key_path_base64_raw: PathBuf,
-    /// Rpc url for Sui fullnode, used for query stuff and submit transactions.
-    pub sui_rpc_url: String,
+pub struct EthConfig {
     /// Rpc url for Eth fullnode, used for query stuff.
     pub eth_rpc_url: String,
-    /// The eth contract addresses (hex). It must not be empty. It serves two purpose:
-    /// 1. validator only signs bridge actions that are generated from these contracts.
-    /// 2. for EthSyncer to watch for when `run_client` is true.
-    pub eth_addresses: Vec<String>,
+    /// The proxy address of SuiBridge
+    pub eth_bridge_proxy_address: String,
+    /// The expected BridgeChainId on Eth side.
+    pub eth_bridge_chain_id: u8,
```

Comment:
wonder if we should remove all `eth` and `sui` prefix to all field names given the struct is now specific to either Eth or Sui. 
e.g. `config.eth.chain_id` instead of `config.eth.eth_bridge_chain_id` etc.